Formula/befunge93.rb

```diff
@@ -0,0 +1,18 @@
+class Befunge93 < Formula
+  desc "Esoteric programming language"
+  homepage "http://catseye.tc/projects/befunge93/"
```

Comment:
I can't reach that URL; but it looks like http://catseye.tc/node/Befunge-93.html works, and it's what's linked to from the GitHub repository.